packages/apps/docs/src/docs/deploy/chainweb/chainweb-get-started.md

```diff
@@ -22,13 +25,487 @@ Before installing Chainweb, you should verify that your computer meets the follo
+For example, you might need to configure port forwarding for port 1789 on your router or host computer to allow inbound connections from remote nodes.
+
 ## Installation options
 
 There are several options for setting up a Chainweb node in a physical or virtual environment.
 For example, you can run a Chainweb node image in a Docker container or build the binaries directly from the source code.
+You can install release binaries directly on a physical Linux server or run them using the infrastructure from a cloud services provider.
+
+For more information about your installation options, see the following topics:
+
+- [Install release binaries]()
+- [Run in a Docker container]()
+- [Build from source code]()
+
+## Install release binaries
+
+You can download compressed archive files with `chainweb-node` release binaries for Ubuntu Linux directly from the [Releases](https://github.com/kadena-io/chainweb-node/releases) pages in the [chainweb-node](https://github.com/kadena-io/chainweb-node/) repository.
+If you have Ubuntu 20.0.4 or Ubuntu 22.04 on a physical or virtual machine, downloading the binary is the most straightforward installation path.
+
+To install from a release archive:
+
+1. Open a terminal shell on the physical or virtual host with the Ubuntu Linux operating system.
+
+2. Update the system with the latest software by running the following command:
+   
+   ```bash
+   sudo apt update && sudo apt upgrade
+   ```
+
+3. Install the required packages by running the following command:
+   
+   ```bash
+   sudo apt-get install ca-certificates libgmp10 libssl3 libsnappy1v5 zlib1g liblz4-1 libbz2-1.0 libgflags2.2 zstd
+   ```
+
+   Note that Ubuntu 20.04 requires the `libssl1.1` package instead of the `libssl3` package. 
+   This package has been deprecated in Ubuntu 22.04.
+   If your computer uses the Ubuntu 20.04 operating system, install the `libssl1.1` package instead of the `libssl3` package. 
+
+4. Download the archive file from the [Releases](https://github.com/kadena-io/chainweb-node/releases) page.
+   
+   Note that the archive file naming convention includes the `chainweb-node` version, compiler version, Ubuntu version, and a commit hash identifier using the following format:
+   
+   ```text
+   chainweb-<version>.ghc-<version>.ubuntu-<version>.<revision>.tar.gz
+   ```
+   
+   For example, the `chainweb-node` archive file for Ubuntu 22.04 looks like this:
+   
+   ```text
+   chainweb-2.24.1.ghc-9.6.5.ubuntu-22.04.89b0ac3.tar.gz
+   ```
+
+5. Change to the directory that contains the downloaded file—typically, the Downloads folder—or move the file to a new location.
+
+6. Unzip the compressed archive by running a command similar to the following:
+   
+   ```bash
+   gunzip chainweb-2.24.1.ghc-9.6.5.ubuntu-22.04.89b0ac3.tar.gz
```

Comment:
you can not use gunzip and just use tar, it has a flag for extracting from .tar.gz: `tar -xvzf`.